src/modules/auth/templates/welcome.pug

```diff
@@ -0,0 +1,93 @@
+doctype html
```

Comment:
not sure what is it, if email, move to some email templates folder

Reply:
yes, this is ".pug" template for sending confirmation link. I placed it in /template folder in Auth module. should I place it somewhere else? For me it looks logically, Auth module uses EmailSender and email templates can be placed right here.